src/patternfly/components/Drawer/drawer.scss

```
@@ -321,6 +331,11 @@ $pf-v6-c-drawer__panel--list--width: (25, 33, 50, 66, 75, 100);
   overflow: auto;
   visibility: hidden; // hidden by default
   background-color: var(--#{$drawer}__panel--BackgroundColor);
+  border: var(--#{$drawer}__panel--BorderWidth) solid var(--#{$drawer}__panel--BorderColor);
```

Since this is only set/used here, and we have border-widths for all of the sides that will override this, would this ever get used? 

```suggestion
  border: solid var(--#{$drawer}__panel--BorderColor);
```